test/testcafe/spec/DeviceChallengePollView_spec.js

```diff
@@ -49,6 +49,11 @@ test
       record => record.response.statusCode === 200 &&
       record.request.url.match(/introspect|6512/)
     )).eql(3);
+    await t.expect(logger.count(
+      record => record.response.statusCode === 200 &&
+      record.request.url.match(/challenge/) &&
+      record.request.body.match(/challengeRequest":"eyJraWQiOiI1/)
```

Comment:
are we able to run `equal` instead of match ?

Reply:
The challengeRequest is very long.

Reply:
what if you don't enable `stringifyRequestBody`, would `request.body` be an simple object  ?
```
{ challengRequest: xxxx }
```

Reply:
It will be a [Buffer](https://nodejs.org/api/buffer.html).